tests/unit/interpreter.cpp

```diff
@@ -1268,6 +1268,23 @@ TYPED_TEST(InterpreterTest, ExecutionStatsValues) {
   }
 }
 
+TYPED_TEST(InterpreterTest, ExecutionStatsValuesPropertiesSet) {
+  {
+    auto [stream, qid] = this->Prepare(
+        "CREATE (u:Employee {Uid: 'EMP_AAAAA', FirstName: 'Bong', LastName: 'Revilla'}) RETURN u.name AS name;");
+    this->Pull(&stream);
+  }
+  {
+    auto [stream, qid] = this->Prepare(
+        "MATCH (node:Employee) WHERE node.Uid='EMP_AAAAA' SET node={FirstName: 'James', LastName: 'Revilla', Uid: "
+        "'EMP_AAAAA', CreatedOn: 'null', CreatedBy: 'null', LastModifiedOn: '1698226931701', LastModifiedBy: 'null', "
+        "Description: 'null'};");
+    this->Pull(&stream);
+    auto stats = stream.GetSummary().at("stats").ValueMap();
+    ASSERT_EQ(stats["properties-set"].ValueInt(), 8);
```

Comment:
Does it make sense to include header `metadata.hpp `to get key from ExecutionStatsKeyToString function for `properties-set`

Reply:
yes